src/main/java/com/happeningnow/service/IBaseService.java

```diff
@@ -3,9 +3,9 @@
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.PageRequest;
 
-public interface IBaseService<T, UUID> {
+public interface IBaseService<T, U> {
```

Comment:
you can use K rather than U so we know that T is for TYPE and K is for KEY

Reply:
OK